### PR TITLE
Assignment 7n8

### PR DESCRIPTION
**Pull request by ap221882**
Added nested routes and error route to the Avengers App using react-router v6

### CODE REVIEW

File: Day7n8/github-api-project-ts/src/containers/Header/Header.tsx [hunk 1]
```diff
@@ -0,0 +1,17 @@
...
+import classes from "./header.styles.module.css";
+import ClassBasedComponent1 from "../../miscellaneous/classBasedComponent1";
+
+type Props = {};
+
+const Header = (props: Props) => {
+  return (
+    <section className={classes.banner__container}>
+      <div className={classes.team__info}></div>
+      <ClassBasedComponent1 />
```
Comment (roshantrivedi) [inline Day7n8/github-api-project-ts/src/containers/Header/Header.tsx]: Numbers can be avoided in naming

File: Day7n8/github-api-project-ts/src/miscellaneous/classBasedComponent1.tsx [hunk 2]
```diff
@@ -0,0 +1,60 @@
...
+ */
+
+/**
+ * * Three react component lifecycles are -
+ * * 1. Mounting - cycle ends with componentDidMount
+ * * 2. Updating - cycle ends with componentDidUpdate
+ * * 3. Unmounting - cycle ends with componentWillUnmount
+ */
+
+class classBasedComponent1 extends Component<Props, State> {
```
Comment (roshantrivedi) [inline Day7n8/github-api-project-ts/src/miscellaneous/classBasedComponent1.tsx]: class name should start with Caps

**Comment by roshantrivedi** [comment]
Reviewed, nicely coded & well written comments as well.

File: Day7n8/github-api-project-ts/src/miscellaneous/classBasedComponent1.tsx (hunk 2 above)
Comment (ap221882) [reply Day7n8/github-api-project-ts/src/miscellaneous/classBasedComponent1.tsx]: Done

File: Day7n8/github-api-project-ts/src/containers/Header/Header.tsx (hunk 1 above)
Comment (ap221882) [reply Day7n8/github-api-project-ts/src/containers/Header/Header.tsx]: Done

File: Day7n8/github-api-project-ts/src/index.tsx [hunk 3]
```diff
@@ -34,9 +34,9 @@ const myAppRouter = createBrowserRouter([
 ]);
 
 root.render(
-  <React.StrictMode>
-    <RouterProvider router={myAppRouter} />
-  </React.StrictMode>
+  // <React.StrictMode>
```
Comment (shaksham08) [inline Day7n8/github-api-project-ts/src/index.tsx:37]: We can remove this commented code if you don't need it for your documenting purpose.

**Pull request merged**